cal/cal/static/js/services.js

```diff
@@ -238,10 +238,11 @@ analyticsApp.service('QueryService', function() {
     var ctrlDetails = [];
     var maxYValue = 0;
     var events = [];
+    var movingAverage = [];
     var xLabels = [];
     var yLabels = [];
-    for (var i = 0; i < data.length; i++) {
-      var event = data[i];
+    for (var i = 0; i < data[0].length; i++) {
```

Comment:
You mean i < data[0].length, j < data[1].length right? So one loop will stop while the other will keep going if they're different lengths?